pyproject.toml

```diff
@@ -7,3 +7,30 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [tool.setuptools_scm]
+
+[tool.coverage.run]
+branch = true
+source = [
+    "src/openscm_runner"
+]
+omit = [
+    "*_version.py",
+    # TODO: remove the exclusion for MAGICC
```

Comment:
Can this be removed now that we've added concurrency?

Reply:
I'll check

Reply:
Turns out it can